baselines/fedper/fedper/run_figures.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# CIFAR10 Mobile and Resnet (non-iid n classes (FIGURE 2a&b))
+for model in mobile resnet
+do 
+    for num_classes in 4 8 10
+    do
+        for algorithm in fedper fedavg
+        do
+           python -m fedper.main --config-path conf --config-name cifar10 dataset.num_classes=${num_classes} model_name=${model} algorithm=${algorithm}
+        done
+    done
+done
+
+
+# CIFAR10 Mobile (n head layers (FIGURE 4a))
+for num_head_layers in 2 3 4
+do
+    python -m fedper.main --config-path conf --config-name cifar10 model.num_head_layers=${num_head_layers} num_rounds=25 model_name=mobile algorithm=fedper&&
+    python -m fedper.main --config-path conf --config-name cifar10 num_rounds=25 model_name=mobile
```

Comment:
I might have mentioned this previously but, isn't this experiment (i.e. not varying `num_head_layers`) being repeated 3 times for no reason? i.e. shouldn't the above be re-written as:
```
# CIFAR10 Mobile (n head layers (FIGURE 4a))
for num_head_layers in 2 3 4
do
    python -m fedper.main --config-name cifar10 model.num_head_layers=${num_head_layers} num_rounds=25 model_name=mobile algorithm=fedper
done
python -m fedper.main --config-name cifar10 num_rounds=25 model_name=mobile
```

(the same applies to the for loops below.)